dev/lib/product_taxonomy/cli.rb

```diff
@@ -95,5 +95,11 @@ def add_attributes_to_categories(attribute_friendly_ids, category_ids)
     def add_value(name, attribute_friendly_id)
       AddValueCommand.new(options.merge(name:, attribute_friendly_id:)).run
     end
+
+    desc "compare_categories VERSION_FOLDER", "Compare category changes between full_names.yml and categories.txt"
```

Comment:
I think we can make the description clearer, maybe something like this:

```suggestion
    desc "compare_categories VERSION_FOLDER", "Generate a CSV report showing category changes between the current taxonomy and Shopify taxonomy VERSION"
```